docs/developers/index.md

```diff
@@ -252,10 +252,10 @@ The Ethereum network is made up of many nodes who run compatible client software
 - [parity.io](https://www.parity.io/)
 - [Github](https://github.com/paritytech/parity-ethereum)
 
-### Besu (Pantheon) *Ethereum client written in Java*
+### Hyperledger Besu (Formely Pantheon) *Ethereum client written in Java*
 - [pegasys.tech](http://pegasys.tech)
-- [Github](https://github.com/PegaSysEng/pantheon/)
-- [Gitter](https://gitter.im/PegaSysEng/pantheon)
+- [Github](https://github.com/hyperledger/besu)
+- [Rocket.Chat](https://chat.hyperledger.org/channel/besu)
```

Comment:
Can you please confirm this link is correct? This URL hangs for me when I try to access it.

Reply:
@samajammin thanks for checking! This is indeed the right link. You need to be logged in to see the conversation. We've already raised an issue with Rocket.Chat about this: https://github.com/RocketChat/Rocket.Chat/issues/15391

Reply:
Bummer. Definitely a suboptimal UX. Any thoughts on how we can improve/avoid that for users? Perhaps link to the chat home (https://chat.hyperledger.org/) instead & mention the #besu channel? Or wait until that issue is resolved?